app/controllers/overrides_controller.rb

```diff
@@ -21,13 +23,22 @@ def create
     @recommended_pom = ResponsibilityService.calculate_responsibility(@prisoner)
     @pom = PrisonOffenderManagerService.get_pom(
       active_caseload, override_params[:nomis_staff_id])
+    @complex_label = complex_reason_label
 
     render :new
   end
 # rubocop:enable Metrics/MethodLength
 
 private
 
+  def complex_reason_label
```

Comment:
Probably a Fixit Friday thing but possibly a bit strange that the controller has view logic mixed in?

Reply:
Yeah, I guess this might be a presenter thing?  It's because this value is used in an `input_tag` so it seemed as messy to try and have a giant if there.  I'll add a fixit friday ticket...